Documentation/CRDs/Object-Storage/ceph-object-store-user-crd.md

```diff
@@ -60,3 +60,26 @@ spec:
     * `user-policy`
     * `odic-provider`
     * `ratelimit`
+
+### CephObjectStoreUser Reference Secret(optional)
+
+User can create a k8s secret where S3 key is user defined.
```

Comment:
```suggestion
If a specific user key and secret is desired instead of randomly generated credentials, a specific user key and secret can be specified for an object store user. 
```